src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -359,6 +359,77 @@ void assertAttributeData(string name)
             }
         }
 
+        [Fact]
+        public void Span_SingleElement_TempsAreNotReused()
```

Comment:
Did we file a bug to follow up on this? This is a place where future optimizations could very much benefit. 

In this particular case the win is pretty minor. It's just an extra `int` or two on the stack. Consider though when we're passing instead 3 parameters and storing lots of redundant inline array temporaries. That can start to make the stack frame size increase be noticable and may deserve an optimization pass to do proper reuse.

Reply:
#73269